packages/next/src/server/next.ts

```diff
@@ -234,12 +234,15 @@ export class NextServer implements NextWrapperServer {
     if (process.env.NODE_ENV === 'production') {
       try {
         const serializedConfig = require(
-          path.join(dir, '.next', SERVER_FILES_MANIFEST)
+          path.join(dir, config.distDir, SERVER_FILES_MANIFEST)
         ).config
 
         // @ts-expect-error internal field
         config.experimental.isExperimentalCompile =
           serializedConfig.experimental.isExperimentalCompile
+        // @ts-expect-error internal field
+        config.experimental.isTurbopackBuild =
+          serializedConfig.experimental.isTurbopackBuild
       } catch (_) {
         // if distDir is customized we don't know until we
```

Comment:
This comment seems outdated since we try to `loadConfig` regardless in the lines right above this try/catch. The require can still fail but likely for other reasons?